improver/cli/snow_fraction.py

```diff
@@ -58,4 +58,6 @@ def process(*cubes: cli.inputcube, model_id_attr: str = None):
     from improver.precipitation_type.snow_fraction import SnowFraction
     from iris.cube import CubeList
 
+    from improver.snow_fraction import SnowFraction
```

Comment:
Failure in tests linked to this - I think it might need swapping with line 58?

Reply:
swapped

Reply:
Ah sorry, my clumsy wording - I meant that the line `improver.snow_fraction` should be replaced with `improver.precipitation_type.snow_fraction` as that's the new location

Reply:
My bad I missed this.

Fixed